src/main/java/org/opentripplanner/middleware/triptracker/TripInstruction.java

```diff
@@ -129,8 +119,6 @@ public String build() {
                 return buildOnTrackInstruction();
             case DEVIATED:
```

Comment:
@br648  I don't feel like touching the walk instructions code for this PR, this could be left for subsequent work, what do you think?

Reply:
@br648 The same way there is a `TransitLegInstruction`, we could do a `WalkLegInstruction` and create subclasses for the different kinds of walk instructions.

Reply:
@binh-dam-ibigroup ok in that case (whilst this is in the forefront of what you are doing) it makes sense to address this now, in this PR.